generators/app/index.js

```diff
@@ -307,6 +307,12 @@ module.exports = class extends BaseGenerator {
                         this.clientPackageManager = 'yarn';
                     }
                 }
+                // Set embeddableLaunchScript to true if not defined, for backward compatibility
+                // See https://github.com/jhipster/generator-jhipster/issues/10255
```

Comment:
double checking - newly generated app will have this set to false by default right?

Reply:
Yes, false for new apps, true for old apps.